apax/md/function_transformations.py

```diff
@@ -96,7 +97,48 @@ def gamd_energy(positions, Z, idx, box, offsets):
         return gamd_energy_force
 
 
+@dataclasses.dataclass
+class GlobalCalibration:
+    """
+    Applies a global calibration to energy and force uncertainties.
+    Energy ensemble predictions are rescaled according to EQ 7 in
+    https://doi.org/10.1063/5.0036522
+
+    Parameters
+    ----------
+    energy_factor : float
+        Global calibration factor by which to scale the energy uncertainty.
+    forces_factor : float
+        Global calibration factor by which to scale the force uncertainties.
+    """
+
+    energy_factor: float
+    forces_factor: float
+
+    def apply(self, model):
+        def calibrated_model(positions, Z, idx, box, offsets):
+            results = model(positions, Z, idx, box, offsets)
+
+            results["energy_uncertainty"] = (
+                results["energy_uncertainty"] * self.energy_factor
+            )
+
+            Emean = results["energy"]
+            Ei = results["energy_ensemble"]
+            results["energy_ensemble"] = Emean + self.energy_factor * (Ei - Emean)
+
+            if "forces_uncertainty" in results.keys():
+                results["forces_uncertainty"] = (
+                    results["forces_uncertainty"] * self.forces_factor
+                )
```

Comment:
Maybe these keys could be imported from some class that lists all keys used by Apax and also fix https://github.com/apax-hub/apax/issues/308

Doesn't have to be in this PR but should be done soon.

Reply:
For what purpose would you import them? Yeah the fix is coming.

Reply:
Ah I guess, we could have an ENERGY_KEY imported from somewhere, yeah but I feel like this is not necessary